argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enable-support-for-namespace-scoped-argo-rollouts-installation"]
+= Enable support for namespace-scoped Argo Rollouts installation
+:context: enable-support-for-namespace-scoped-argo-rollouts-installation
+
+toc::[]
+The {gitops-title} enables support for two modes of Argo Rollouts installation:
+
+* *Cluster-scoped installation* (default): The Argo Rollouts custom resources (CRs) defined in any namespace are reconciled by the Argo Rollouts instance. This allows you to use Argo Rollouts CR across any namespace on the cluster.
+* *Namespace-scoped installation*: The Argo Rollouts instance is installed in a specific namespace and only handles Argo Rollouts CR within the same namespace. This installation mode includes the following benefits:
+** This mode does not require cluster-wide `ClusterRole` or `ClusterRoleBinding` permissions. This means that only users with permissions that are limited to a specific namespace can use these resources.
```

Comment:
'can use these resources' -> 'can use Argo Rollouts resources'. 

At present, 'these' in the sentence seems like it could be referring to CR/CRB.

Or, if you prefer, another way of doing it:
'This mode does not require cluster-wide `ClusterRole` or `ClusterRoleBinding` permissions. Users may install and use Argo Rollouts within a single namespace without requiring these cluster permissions.'